### PR TITLE
Feature/project card improvemetns

### PR DESCRIPTION
**Pull request by niteshgrg**
------Update Nov 9------
iOS Testing

- [x] Fix issue with FontAwesome

Screen Shot 2019-11-09 at 2 20 11 PM

- [x]  Remove Shadow from project Cards
Screen Shot 2019-11-09 at 2 20 35 PM

- [x] Fix Missing Flag icon
Screen Shot 2019-11-09 at 2 18 46 PM

------Update Nov 7------

**Fixes for the Project Part**

Design is here: https://zpl.io/2yPB70K
- [x] Fix Tabs not working for Android (current build) not sure if that exists in this specific branch.

- [x] Change Page Title to Projects

- [x] Show country as Location from the coordinates in the card preview.
Screen Shot 2019-11-07 at 11 38 20 AM

- [x] In Single Page, Make  the button clickable:
Screen Shot 2019-11-07 at 11 39 12 AM

- [x] Remove shadow and border from cards

Screen Shot 2019-11-07 at 11 42 20 AM

- [x] In the my trees part just fix this part for now:

Screen Shot 2019-11-07 at 1 02 44 PM

------Original -------
This PR continues the work or PR #1298

It should implement #873 and fix the bug #950

### CODE REVIEW

**Comment by mahmed0715** [comment]
@niteshgrg what is the plan for this PR? what issues it should solve @norbertschuler @sagararyal ?

File: app/components/PlantProjects/PlantedProgressbar.native.js [hunk 1]
```diff
@@ -4,16 +4,26 @@ import PropTypes from 'prop-types';
...
 import styles from '../../styles/selectplantproject/selectplantproject-snippet.native';
-import { targetPlanted } from '../../assets';
-import { delimitNumbers } from '../../utils/utils';
+import { challenge_outline as targetPlanted } from '../../assets';
+import { delimitNumbers, convertNumber } from '../../utils/utils';
+import EStyleSheet from 'react-native-extended-stylesheet';
+
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+const flagIcon = <Icon name="flag" size={18} color="#ffffff" />;
```
Comment (norbertschuler) [inline app/components/PlantProjects/PlantedProgressbar.native.js]: The new module `react-native-vector-icons` and fonts seem to be only imported for this icon.

File: app/components/PlantProjects/PlantProjectSnippet.native.js [hunk 2]
```diff
@@ -3,14 +3,18 @@ import React, { PureComponent } from 'react';
...
 import i18n from '../../locales/i18n';
 import styles from '../../styles/selectplantproject/selectplantproject-snippet.native';
 import { formatNumber } from '../../utils/utils';
 import PrimaryButton from '../Common/Button/PrimaryButton';
 import CardLayout from '../Common/Card';
 import PlantedProgressBar from './PlantedProgressbar.native';
 
+import Icon from 'react-native-vector-icons/FontAwesome';
+//keeping Icon here instead of in assets
+const starIcon = <Icon name="star" size={14} color="#4d5153" />;
```
Comment (norbertschuler) [inline app/components/PlantProjects/PlantProjectSnippet.native.js]: The new module react-native-vector-icons and fonts seem to be only imported for this icon and one other icon.

File: ios/Podfile [hunk 3]
```diff
@@ -3,4 +3,6 @@
 
 target 'TreecounterApp' do
   inherit! :search_paths
+  pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'
```
Comment (norbertschuler) [inline ios/Podfile]: Before we merge this, I would like to remove this and import the project manually to the iOS project as described e.g. here: https://github.com/oblador/react-native-vector-icons.

**Comment by mahmed0715** [comment]
@sagararyal what else left in this PR according to the actual requirement ticket? Please let me know.
I will clean up code and adapt with new changes for number format and other.
@norbertschuler please check

File: ios/TreecounterApp.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -56,6 +55,22 @@
...
+		F27C38C01A514DC0B4AF444D /* Fontisto.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1D12490D15E24C07B02AE93C /* Fontisto.ttf */; };
+		03D4B0E14CCD461F8D6E6F7F /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 29C5C3FAB731455CB2EB86BB /* Foundation.ttf */; };
+		8B431965E3C6427C8E686057 /* Ionicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = B809826C80D54507B21FD780 /* Ionicons.ttf */; };
+		9678EEA4889F48CC87E2DE88 /* MaterialCommunityIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4E7B7230CEBA4B6B8B547072 /* MaterialCommunityIcons.ttf */; };
+		08C65E2A9C2E412FAB60E933 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = A94FFCB5ADD84164A073C9BE /* MaterialIcons.ttf */; };
+		EDCEB8F8FF784BEE856D5511 /* Octicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = A4C19E64E64B4280B54B78FF /* Octicons.ttf */; };
+		45971A74356E495FAD4C30E5 /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 8F9055B5AE734E9E9619EC52 /* SimpleLineIcons.ttf */; };
+		4548A4F8A6CB458FADE56DD6 /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 9FDEFB4763DA4158BBD7D220 /* Zocial.ttf */; };
```
[the remark this answers is not in the document]
Comment (harshvitra) [reply ios/TreecounterApp.xcodeproj/project.pbxproj]: @mahmed0715 considering other features we will need only FontAwesome and MaterialCommunityIcons.

**Comment by mahmed0715** [comment]
@niteshgrg is it possible you please fix ios issue. I have issue with ios emulator.

**Comment by norbertschuler** [comment]
> @niteshgrg is it possible you please fix ios issue. I have issue with ios emulator.

@mahmed0715 I think @niteshgrg left the project, but @sanjayradadiya wants to try fixing the iOS issues by migrating to react-native 0.60.x (or 0.61.x) now.

**Comment by mahmed0715** [comment]
One thing you please confirm that there is no shadow on the project card. if thats fixed we can merge Sagar says

**Comment by mahmed0715** [comment]
I really appreciate this @norbertschuler . Thanks

**Comment by norbertschuler** [comment]
> One thing you please confirm that there is no shadow on the project card. if thats fixed we can merge Sagar says

No, there are still shadows around the projects in the list as well as in the detail view for iOS:

Bildschirmfoto 2019-11-12 um 16 30 36

**Comment by sagararyal** [comment]
awesome 👏 😎

**Pull request merged**